spec/factories/buildings.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+FactoryGirl.define do
+  sequence :building_name do |n|
```

Comment:
As discussed, we're going to inline these sequences inside the actual `factory` block.